saleor/settings.py

```diff
@@ -43,14 +43,9 @@
 EMAIL_URL = os.environ.get('EMAIL_URL', 'console://')
 email_config = dj_email_url.parse(EMAIL_URL)
 
-EMAIL_FILE_PATH = email_config['EMAIL_FILE_PATH']
-EMAIL_HOST_USER = email_config['EMAIL_HOST_USER']
-EMAIL_HOST_PASSWORD = email_config['EMAIL_HOST_PASSWORD']
-EMAIL_HOST = email_config['EMAIL_HOST']
-EMAIL_PORT = email_config['EMAIL_PORT']
-EMAIL_BACKEND = email_config['EMAIL_BACKEND']
-EMAIL_USE_TLS = email_config['EMAIL_USE_TLS']
+vars().update(email_config)
```

Comment:
I have gone through the code. 
`EMAIL_USE_SSL` has failed in few cases give KeyErrors. 
Which is the reason, I haven't pushed the commit.

I have sent a new pull request to `dj-email-url` it's better to update after that.